### ISSUE TITLE
[BDM] - Heartbeat

### ISSUE THREAD

**Issue by Peiprjs** [PGamerX-Legacy/issues#12]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Adding a heartbeat function to the bots to detect problems as soon as possible. Maybe make it send a heartbeat per interaction?

Checklist: 
- [x] Black Sheep
- [ ] Muser
- [ ] Domework

**Comment by pgamerx** [comment]
Per interaction will be hectic and might just cause us to hit the discord api rate-limit

**Comment by Peiprjs** [comment]
As far as I know, Cronitor doesn't have ratelimits. Let me read the documentation rq tho

**Comment by Peiprjs** [comment]
Nah, don't look like it. Could be interesting to have some more data.

**Comment by pgamerx** [comment]
> Nah, don't look like it. Could be interesting to have some more data.

I said Discord's API

**Comment by Peiprjs** [comment]
Why would we use DAPI?

**Comment by Peiprjs** [comment]
We have the CRAPI

**Comment by pgamerx** [comment]
> We have the CRAPI

Makes sense, go ahead.

**Comment by pgamerx** [comment]
@Peiprjs we have fixed this right?

**Comment by Peiprjs** [comment]
I just have to implement it, but that's probably a 30 minute job per bot, I can just copy stuff from the other bot.

**Comment by pgamerx** [comment]
k
